hermes/bin/src/ipfs/mod.rs

```diff
@@ -353,8 +361,12 @@ struct AppIpfsState {
     dht_keys: DashMap<ApplicationName, HashSet<DhtKey>>,
     /// List of subscriptions per app.
     topic_subscriptions: DashMap<PubsubTopic, HashSet<ApplicationName>>,
+    /// List of subscriptions per app (Doc Sync).
+    doc_sync_topic_subscriptions: DashMap<PubsubTopic, HashSet<ApplicationName>>,
     /// Collection of stream join handles per topic subscription.
     subscriptions_streams: DashMap<PubsubTopic, JoinHandle<()>>,
+    /// Collection of stream join handles per topic subscription (Doc Sync).
+    doc_sync_subscriptions_streams: DashMap<PubsubTopic, JoinHandle<()>>,
```

Comment:
```suggestion
    doc_sync_topic_subscriptions: DashMap<PubsubTopic, StructWithHashSetAndHandler>,
    /// Collection of stream join handles per topic subscription.
    subscriptions_streams: DashMap<PubsubTopic, JoinHandle<()>>,
```
Does such change make sense? Or there is a reason to have multiple dash maps?

Reply:
For a single topic we can have both doc sync subscription and regular ipfs subscription.
The kind would have to go as a key, but it's less of an intrusive change with multiple maps, I think.